trinity/plugins/builtin/ethstats/ethstats_client.py

```diff
@@ -0,0 +1,141 @@
+import asyncio
+import datetime
+import json
+import typing
+
+import websockets
+
+from p2p.service import (
+    BaseService,
+)
+
+
+# Returns UTC timestamp in ms, used for latency calculation
+def timestamp_ms():
+    return round(datetime.datetime.utcnow().timestamp() * 1000)
+
+
+class EthstatsMessage(typing.NamedTuple):
+    command: str
+    data: dict
+
+
+class EthstatsException(Exception):
+    pass
+
+
+class EthstatsClient(BaseService):
+    def __init__(
+        self,
+        websocket: websockets.client.WebSocketClientProtocol,
+        node_id: str,
+        *args,
+        **kwargs,
+    ) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.websocket = websocket
+        self.node_id = node_id
+
+        self.send_queue: asyncio.Queue = asyncio.Queue()
```

Comment:
I think this needs to be typed as `'asyncio.Queue[EthstatsMessage]'`

Reply:
Fixed this.